### PR TITLE
Undoes the Syringe Infections

### PR DESCRIPTION
**Pull request by Mewchild**
As per: https://github.com/VOREStation/VOREStation/issues/3972#issuecomment-403142802
Reverts the changes that give syringes the chance to infect
Reverts Syringe boxes to hold 7 needles per box, as it used to be

All code work was done by @LiquidFirefly

### CODE REVIEW

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -203,28 +202,26 @@
...
+				add_attack_logs(user,target,"Injected with [src.name] containing [contained], trasferred [trans] units")
 			else
 				to_chat(user, "<span class='notice'>The syringe is empty.</span>")
 
-			dirty(target,affected) //VOREStation Add
 
 	return
-/* VOREStation Edit - See syringes_vr.dm
+
 /obj/item/weapon/reagent_containers/syringe/update_icon()
```
Comment (Arokha) [inline code/modules/reagents/reagent_containers/syringes.dm]: Why do you want to revert to the old update_icon? The new one is far more efficient and actually makes the new icons work. Did you test this...? Even if you did, "no", you can't use overlays.Cut() and stuff anymore, you need to use the functions so things don't break.

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 2]
```diff
@@ -203,28 +202,26 @@
...
 
 			if(trans)
 				to_chat(user, "<span class='notice'>You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units.</span>")
-				if(ismob(target))
-					add_attack_logs(user,target,"Injected with [src.name] containing [contained], trasferred [trans] units")
+				add_attack_logs(user,target,"Injected with [src.name] containing [contained], trasferred [trans] units")
 			else
 				to_chat(user, "<span class='notice'>The syringe is empty.</span>")
 
-			dirty(target,affected) //VOREStation Add
```
Comment (Arokha) [inline code/modules/reagents/reagent_containers/syringes.dm]: This is the only line you need to alter. The rest of the stuff is totally unrelated to syringe infections and will just fuck up all the icons.

**Comment by LiquidFirefly** [comment]
It reverts syringes entirely to how they were before. That means no extra clicks from caps, and a change back to the old icons.   This was appropriated from something that I did in my free time simply to sit there and see how long it would actually take to revert this, not thinking it'd ever get reverted.  If it was a single line, why was this not done literally a month ago when it was announced it should be changed?

**Comment by Arokha** [comment]
Dunno, if you have free time, why didn't you make a PR to change that line?

**Comment by LiquidFirefly** [comment]
This had cursory testing, syringes looked fine injecting and taking in reagents.

**Comment by Arokha** [comment]
I get that it looks fine, but you don't understand what I'm saying. The non-SSoverlays method of messing with overlays is obsolete and shouldn't be used anymore. We (and tg and everyone else) have switched to using an overlays subsystem that has specific macros for dealing with overlays, as shown in the other file. You need to stick with that.

**Comment by LiquidFirefly** [comment]
Perhaps it's because I don't pay that much attention anymore.   It's not my self appointed _job_ to sit here and make changes to the server, and make sure that changes that are supposed to happen, happen.   It's someone elses.

**Comment by Arokha** [comment]
Well, it's nobody's self-appointed job other than Ace's, really. Everyone else was appointed by others!

Doesn't matter, anyway. Needs to be fixed so it uses SSoverlays and doesn't re-implement old overlays.Cut() stuff lest we get back to crashy-server-times.

**Comment by Mewchild** [comment]
I’ll make the suggested changes when I’m home from work

**Comment by SpadesNeil** [comment]
image

Actually it's nobody's _**job**_ to do any of this. We donate our time and effort to something we enjoy. LiquidFirefly, if all you're here to do is bitch, then you can go somewhere else.

Mewchild, you keep up the good work and we'll provide help where we can.

**Comment by Mewchild** [comment]
Hey now. Let’s keep it civil and chill. Don’t want these bad vibes in my email from y’all

**Comment by Mewchild** [comment]
Remaking this PR with the recommended changes.

**Comment by SpadesNeil** [comment]
Sounds fine to me. Will lock this old thread.

**Pull request closed without merging**